api.md

```diff
@@ -469,7 +535,66 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
 
+Detach a data node from a hypertable or from all hypertables.
+
+This function is typically used if a data node should no longer be
+used. Then it need to be removed from all hypertables that use
+it. Another reason for detaching a data node from a hyper table is
+that you want to have fewer
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name              | Description                                   |
+| `node_name`       | Local name of data node to attach             |
+
+##### Parameter Description
+
+* `node_name` - Local name of the data node to attach to the
+  hypertable.
```

Comment:
Local name of the data node to detach from the hypertable?

Reply:
Fixed (and the detailed description below).